### ISSUE TITLE
Tellurium not working with Windows non-ascii user names

### ISSUE THREAD

**Issue by matthiaskoenig** [sys-bio/tellurium#121]
Hi all,
I just wanted to try the windows version of tellurium and installed tellurium it in a Windows 7 virtual machine and run into the following troubles:

When clicking on the icon the the spyder icon is shown and nothing more happens. The icon vanishes and than nothing is happening. Clicking a second time I get the window "Spyder crashed during the last session."

When clicking directly on the `Spyder.exe` in the install folder the spyder GUI is starting, but I do not have any kernel. I get the message
"An error ocurred while starting the kernel"

And on the ipython console

```
[IPKernelApp] CRITICAL | Bad config encountered during initialization:
[IPKernelApp] CRITICAL | Could not decode 'C:\\Users\\Matthias K\xf6nig\\AppData\\Roaming\\jupyter\\runtime' for unicode trait 'connection_dir' of an IPKernelApp instance.
```

After some research it seems there are some issues with ipython (python 2) and non-ascii Windows user names, like reported here
https://github.com/jupyterhub/jupyterhub/issues/444

Workaround:
Create a new user account only containing ascii characters and run the `Spyder.exe` from the respective user account as administrator.
No solution required, only for information. This should be solved with migration to python 3 at some point.

Matthias

**Comment by matthiaskoenig** [comment]
Just to add:
After creating the new user name the user folder in the tellurium desktop link has to be changed to the ascii-user folder. There seems to be a second issue with Spyder.exe being called with -w and non-ascii user names. So the link must be updated also.

**Comment by kirichoi** [comment]
This issue is fixed on Spyder IDE side since version 3.0.2 apparently.
